src/components/layout/Navbar.js

```diff
@@ -0,0 +1,264 @@
+import { NavLink, Link } from 'react-router-dom';
+import { useEffect, useRef, useState } from 'react';
+import { Transition } from '@headlessui/react';
+import useOnClickOutside from 'use-onclickoutside';
+
+import '../../styles/layout/navbar.css';
+
+export default function Navbar() {
+  const [isMenuShowing, setIsMenuShowing] = useState(false);
+  const [isMobileMenuShowing, setisMobileMenuShowing] = useState(false);
+  const menuRef = useRef(null);
+
+  useOnClickOutside(menuRef, () => setIsMenuShowing(false));
+
+  return (
+    <nav className="bg-primary">
+      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+        <div className="flex items-center justify-between h-16">
+          <div className="flex-shrink-0">
+            <h1 className="text-brand">Mybid</h1>
+          </div>
+          <div className="hidden md:block">
+            <div className="ml-10 flex items-baseline space-x-4">
+              <NavLink exact to="/" className="navlink">
+                Beranda
+              </NavLink>
+
+              <NavLink to="/objeklelang" className="navlink">
```

Comment:
URL nya mngkin bisa diubah biar lebih jelas, contohnya
```suggestion
              <NavLink to="/lelang/objek" className="navlink">
```